android/build.gradle

```diff
@@ -97,6 +97,6 @@ dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'com.github.software-mansion:react-native-executorch:main-SNAPSHOT'
```

Comment:
I think there should be no change here.

Reply:
In my opinion on main we should have a version of library that have every feature working. As for now we don't want to make a new release for android lib as it can change before 0.2.0, that's why it's linked locally.